api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher.csproj

```diff
@@ -7,6 +7,7 @@
     <Description>Azure Industrial IoT OPC Publisher API</Description>
   </PropertyGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\..\components\opc-ua\src\Microsoft.Azure.IIoT.OpcUa\src\Microsoft.Azure.IIoT.OpcUa.csproj" />
```

Comment:
We should only take dependencies on the API assemblies here, not to internal ones.